modern/src/common/converter.js

```diff
@@ -0,0 +1,24 @@
+const speedConverter = (value, unit) => {
+  let factor;
+  switch (unit) {
+    case 'kmh':
+      factor = 1.852;
+    case 'mph':
+      factor = 1.15078;
+    case 'kn':
+    default:
+      factor = 1;
+  }
+  return (value * factor).toFixed(2);
+};
+
+export const getConverter = (key) => {
```

Comment:
I don't think we need this. Please just export separate functions instead.

Reply:
done